src/elements/form/Control.ts

```diff
@@ -28,90 +28,103 @@ import { Helpers } from './../../utils/Helpers';
+                            <!--QuickNote-->
+                            <novo-quick-note *ngSwitchCase="'quick-note'" [formControlName]="control.key" [placeholder]="control.placeholder" [config]="control.config" (change)="modelChange($event)"></novo-quick-note>
+                             <!--ReadOnly-->
+                            <!--TODO - Handle rendering of different READONLY values-->
+                            <div *ngSwitchCase="'read-only'">{{ form.value[control.key] }}</div>
+                        </div>
                     </div>
-                    <!--TextArea-->
-                    <textarea *ngSwitchCase="'text-area'" [name]="control.key" [attr.id]="control.key" [placeholder]="control.placeholder" [formControlName]="control.key" (input)="resizeTextArea($event)" (input)="emitChange($event)" [maxlength]="control.maxlength"></textarea>
-                    <!--Editor-->
-                    <novo-editor *ngSwitchCase="'editor'" [name]="control.key" [formControlName]="control.key" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></novo-editor>
-                    <!--HTML5 Select-->
-                    <select [id]="control.key" *ngSwitchCase="'native-select'" [formControlName]="control.key">
-                        <option *ngIf="control.placeholder" value="" disabled selected hidden>{{control.placeholder}}</option>
-                        <option *ngFor="let opt of control.options" [value]="opt.key">{{opt.value}}</option>
-                    </select>
-                    <!--File-->
-                    <novo-file-input *ngSwitchCase="'file'" [formControlName]="control.key" [id]="control.key" [name]="control.key" [placeholder]="control.placeholder" [multiple]="control.multiple"></novo-file-input>
-                    <!--Tiles-->
-                    <novo-tiles *ngSwitchCase="'tiles'" [options]="control.options" [formControlName]="control.key" (onChange)="modelChange($event)"></novo-tiles>
-                    <!--Picker-->
-                    <div class="novo-control-input-container" *ngSwitchCase="'picker'">
-                        <novo-picker [config]="control.config" [formControlName]="control.key" [placeholder]="control.placeholder" [appendToBody]="control.appendToBody" [parentScrollSelector]="control.parentScrollSelector" *ngIf="!control.multiple" (select)="modelChange($event);" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></novo-picker>
-                        <chips [source]="control.config" [type]="control.config.type" [formControlName]="control.key" [placeholder]="control.placeholder" *ngIf="control.multiple" [closeOnSelect]="control.closeOnSelect" (changed)="modelChange($event)" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></chips>
+                    <!--Error Message-->
+                    <div class="field-message" *ngIf="errors || showCount || control.description || !condensed">
```

Comment:
Condensed should win here, everything on the Forms look good, but the controls in the Tables use condense to hide all messages under the control. They appear now.

<img width="914" alt="screenshot 2017-02-07 12 03 25" src="https://cloud.githubusercontent.com/assets/870799/22702406/96d9bd5a-ed2e-11e6-9c02-0803ab00a122.png">

```
<td *ngFor="let column of columns" [attr.data-automation-id]="column.id || column.name" [class.novo-form-row]="editable">
  <novo-table-cell *ngIf="!row._editing[column.name]" [hasEditor]="editable" [column]="column" [row]="row" [form]="tableForm.controls.rows.controls[i]"></novo-table-cell>
  <novo-control *ngIf="row._editing[column.name]" condensed="true" [form]="tableForm.controls.rows.controls[i]" [control]="row.controls[column.name]"></novo-control>
</td>
```